From: Andreas Pehrson <apehrson@mozilla.com>
Date: Wed, 18 Oct 2023 17:21:00 +0000
Subject: Bug 1859786 - Fix clang-tidy warning in video_capture_impl.cc.
 r=webrtc-reviewers,mjf

clang-tidy says:
  'auto dataCallBack' can be declared as 'auto *dataCallBack'

Differential Revision: https://phabricator.services.mozilla.com/D191302
Mercurial Revision: https://hg.mozilla.org/mozilla-central/rev/26c84d214137a1b0de0902c7038756964e5786f4
---
 modules/video_capture/video_capture_impl.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/video_capture/video_capture_impl.cc b/modules/video_capture/video_capture_impl.cc
index dad8bee1f8..46fff89a52 100644
--- a/modules/video_capture/video_capture_impl.cc
+++ b/modules/video_capture/video_capture_impl.cc
@@ -134,7 +134,7 @@ int32_t VideoCaptureImpl::DeliverCapturedFrame(VideoFrame& captureFrame) {
 
   UpdateFrameCount();  // frame count used for local frame rate callback.
 
-  for (auto dataCallBack : _dataCallBacks) {
+  for (auto* dataCallBack : _dataCallBacks) {
     dataCallBack->OnFrame(captureFrame);
   }
 
